App/Settings.php

```diff
@@ -90,7 +90,21 @@ public function __construct() {
 							'output_processing' => array(
 								'encode_special'
 							)																						
-						]						
+						],
+						'location_precision' => [
+							'type' => 'select',
+							'default' => '10cm',
+							'options' => [
+								'6' => '10cm',
```

Comment:
I suggest instead of the value "6", this should be set to "0". This can be used as our _flag_ to indicate that the value does not need to be modified.